setup.py

```diff
@@ -154,7 +194,8 @@ def build_mgclient_for(self, extension: Extension):
             self.spawn([cmake_binary,
                         '--build', mgclient_build_path,
                         '--config', build_type,
-                        '--target', 'install'])
+                        '--target', 'install',
+                        '--verbose'])
```

Comment:
The verbose output supposed to be temporary only, but the command always run in verbose mode ('self.verbose` is always not `None`), even when the verbosity is turned off with `--quiet`. Therefore I couldn't find a way in which the user can ask for verbose output, as a result the decision about use verbose output always or never. I choose to use it always, as the more is better in this case in my opinion.